_format/1.1/index.md

```diff
@@ -510,7 +538,7 @@ Each member of a links object is a "link". A link **MUST** be represented as
 either:
 
 * a string containing the link's URL.
-* <a id="document-links-link-object"></a>an object ("link object") which can 
+* <a id="document-links-link-object"></a>an object ("link object") which can
```

Comment:
There is no actual change here, just reflowing. This change can/should hence be reverted AFAICT.